packages/backend/src/services/CsvService/CsvService.ts

```diff
@@ -710,6 +711,21 @@ export class CsvService extends BaseService {
             throw new ForbiddenError();
         }
 
+        if (
```

Comment:
how come this has to be declared here too? Isn't there a single run query function to check for permissions.

Reply:
> Isn't there a single run query function to check for permissions.

no :(

Reply:
a lot of work from what you've seen here?

Reply:
yes, a bit. There are multiple chains of functions in project service which I can't keep track of to have the full picture.